crates/bevy_solari/src/pathtracer/pathtracer.wgsl

```diff
@@ -39,28 +41,26 @@ fn pathtrace(@builtin(global_invocation_id) global_id: vec3<u32>) {
         let ray_hit = trace_ray(ray_origin, ray_direction, ray_t_min, RAY_T_MAX, RAY_FLAG_NONE);
         if ray_hit.kind != RAY_QUERY_INTERSECTION_NONE {
             let ray_hit = resolve_ray_hit_full(ray_hit);
-
-            // Evaluate material BRDF
-            let diffuse_brdf = ray_hit.material.base_color / PI;
+            let wo = -ray_direction;
 
             // Use emissive only on the first ray (coming from the camera)
             if ray_t_min == 0.0 { radiance = ray_hit.material.emissive; }
 
             // Sample direct lighting
             let direct_lighting = sample_random_light(ray_hit.world_position, ray_hit.world_normal, &rng);
-            radiance += throughput * diffuse_brdf * direct_lighting.radiance * direct_lighting.inverse_pdf;
+            let direct_lighting_brdf = evaluate_brdf(ray_hit.world_normal, wo, direct_lighting.wi, ray_hit.material);
+            radiance += throughput * direct_lighting.radiance * direct_lighting.inverse_pdf * direct_lighting_brdf;
```

Comment:
Shouldn't this also be multiplied by `cos(theta)`?

Reply:
No, cos(theta) is part of either the throughput for the next bounce, or sample_ramdom_light.